TestIntegration/flash.sh

```diff
@@ -5,6 +5,6 @@ WORKSPACE_DIR=$(realpath "${SCRIPT_DIR}/../")
 
 export ProjDirPath
 ProjDirPath="$(realpath "${WORKSPACE_DIR}/Projects/b_u585i_iot02a_ntz")"
-(cd $ProjDirPath; \
-    BuildArtifactFileBaseName="b_u585i_iot02a_ntz" \
-    "${WORKSPACE_DIR}"/tools/stm32u5_tool.sh flash_ntz)
+cd "${ProjDirPath}" || exit
```

Comment:
Here it changes directory, and not in a subshell. I think this may leave things in a different state then when the script is run and confuse the caller?

You could use a subshell '(...)' or cd to thw starting pwd, etc.

Reply:
Updated to use pushd / popd